### PR TITLE
Adds support for undefined templates

### PR DESCRIPTION
**Pull request by earobinson**
- Also updated some dependencies

### CODE REVIEW

**Comment by misshula** [comment]
!!

File: lib/jyson.js [hunk 1]
```diff
@@ -16,35 +16,43 @@ const getKey = (object, key) => {
...
+  return json[key] = value;
+};
+
+const fillKeys = (json, object, undefinedValue, opts) => {
   const jsonResult = {};
   Object.keys(json).forEach(key => {
     // String Path
     if(_.isString(json[key])){
-      const value = getKey(object, json[key], null);
+      const value = getKey(object, json[key], undefinedValue);
```
Comment (canuc) [inline lib/jyson.js:32]: undefinedValue == valueNotPresentSentinel

File: lib/jyson.js [hunk 2]
```diff
@@ -16,35 +16,43 @@ const getKey = (object, key) => {
   });
 };
 
-const fillKeys = (json, object, opts) => {
+const setValue = (json, key, value) => {
+  if (value === undefined) {
```
Comment (canuc) [inline lib/jyson.js]: typeof value === 'undefined' is safer

**Review by canuc — DISMISSED** [review]
Sick! LGTM two possible improvements

File: lib/jyson.js (hunk 1 above)
Comment (canuc) [reply lib/jyson.js:32]: nvm thats dumb

**Review by misshula — DISMISSED** [review]

**Comment by earobinson** [comment]
pulling @etorrejon off this to add @leighanotleia

**Review by leighanotleia — APPROVED** [review]

**Pull request merged**